api/python/arrus/metadata.py

```diff
@@ -29,16 +30,27 @@ class DataDescription(abc.ABC):
     pass
 
 
+@dataclasses.dataclass(frozen=True)
+class Grid:
+    """
+    N-dimensional grid of coordinates.
+    """
+    coordinates: Sequence[Sequence[float]]
+
+
 @dataclasses.dataclass(frozen=True)
 class EchoDataDescription(DataDescription):
```

Comment:
Maybe USDataDescription or RxDataDescription would be more general - received data not necessarily must be 'an echo'. For example in 'transmission' setup, like in US tomography.

Reply:
I've created issue for metadata refactoring (see https://us4us.myjetbrains.com/youtrack/issue/ARRUS-252). For backward compatibility reasons we will not change the name of this class for now.